res/controllers/Reloop Terminal Mix 2-4.js

```diff
@@ -121,6 +121,11 @@ TerminalMix.samplerVolume = function (channel, control, value) {
     }
 }
 
+TerminalMix.pitchSlider = function (channel, control, value, status, group) {
+	// invert pitch slider (down=faster) so it matches the labels on controller
+	engine.setValue(group,"rate",-script.midiPitch(control, value, status));
+}
+
```

Comment:
Okay, I will split them next time.
midiPitch: the wiki describes a simple implementation like this:
engine.setValue("[Channel"+deck+"]","rate",script.midiPitch(control, value, status));
I don't understand what's wrong. Terminal mix sends messages like this:
status 0xE1: pitch bend ch 2, value 0x2AA7
and everything works perfectly.